{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/static_source/css/base.scss

```diff
@@ -6,19 +6,15 @@
 @import "tailwindcss/lib/css/preflight";
 @import "tailwindcss/base";
 
+@import "base/base";
```

Comment:
This is replacing our app.scsss - I think it should stay in the same spot, I seem to remember some wierdniss about import order